src/models/formFields.ts

```diff
@@ -0,0 +1,17 @@
+export interface FormFields {
+  username: string;
+  name: string;
+  email: string;
+  age: number;
+  skills: any;
+  commitment: number;
+  additionalInfo: string;
+  resumeUrl: string;
+  experience: string;
+  reason: string;
+  accepted: boolean;
+  zip: number;
+  role: string;
+  id: string;
+  project: string;
+}
```

Comment:
there's a field that isn't being used here which is `level`

Reply:
apparently it's doesn't get sent to api when making the request as well
![image](https://user-images.githubusercontent.com/27841994/174668773-8017cda7-7abc-4252-bcf4-415d0f1a0e2d.png)